package-lock.json

```diff
@@ -9909,16 +9909,13 @@
           "resolved": "http://registry.npmjs.org/got/-/got-6.7.1.tgz",
```

Comment:
Could you please reset this file?

Reply:
@aslafy-z do you mean to revert it to one in the master repo?

Reply:
Yep, since you haven't made any updates to the deps, this file shouldn't change 😄